### DISCUSSION TITLE
SD3 Support? pretty please

### DISCUSSION THREAD

**Discussion by Darknessssenkrad** [lllyasviel/stable-diffusion-webui-forge#815] [General]
SD3 Support? pretty please

**Comment by a3nima** [comment]
Is SD3 currently supported by Forge (just asking for informational purpose..) ? tia

**Comment by pflky** [comment]
SD3 is pretty much DOA, especially with the release of Flux. So it wouldn't even really be worthwhile to support it. Lack of LORA/Controlnets/etc. At that point you might as well just use ComfyUI, because that'll have all you need to run it. So it's almost pointless for Forge to support it. You'll have a better time with SDXL + LORAs + Controlnet than you will with SD3.  Even if Forge ever did get SD3 support, most people probably would just use it to try it out and then move on quickly out of disappointment, so you'd honestly just be better off using online SD3 generators, because you're not missing anything by not being able to run it locally.

**Comment by andy8992** [comment]
> SD3 Support? pretty please

This is pretty crazy it's not supported. Stability promised another version, there is now a hyper lora as of today and I can't use the damn thing at all because people think I should be using a 24gb model that drags ass on my 8gb vram pc... stupid

**Reply by pflky** [reply]
> > SD3 Support? pretty please
> 
> This is pretty crazy it's not supported. Stability promised another version, there is now a hyper lora as of today and I can't use the damn thing at all because people think I should be using a 24gb model that drags ass on my 8gb vram pc... stupid

But it's not really crazy, because SD3 is a big flop, and Flux is everything SD3 was supposed to be, so Flux is the superior model. There will simply not be much support for it with things like LORA's and controlnets, so it's not really worth bothering.

**Reply by ZeroCool22** [reply]
> > SD3 Support? pretty please
> 
> This is pretty crazy it's not supported. Stability promised another version, there is now a hyper lora as of today and I can't use the damn thing at all because people think I should be using a 24gb model that drags ass on my 8gb vram pc... stupid

With **SD 3.1** maybe things will change... (maybe).

**Comment by E2GO** [comment]
SD3.5 is out. https://stability.ai/news/introducing-stable-diffusion-3-5
Now could be nice to have support of using it in Forge.

**Reply by rroobbdd33** [reply]
Seems like a reasonable request, might stop people springing over to the spaghetti? SD3.5 looks as though it might be a reasonable base after all... and if it's faster and easier to train... and...

**Reply by evanheckert** [reply]
Yes, I'm really liking what I'm seeing considering it's a base model. Would love to see support in Forge.

**Reply by angelika3377** [reply]
yes please, it looks great for what it is now, especially turbo

**Reply by MrCudou** [reply]
Forge support for 3.5 would be amazing. I expect in the coming weeks/months we'll see quite a lot of refining and LORAs made for it. It may be a very nice evolution replacement for SDXL if all goes well 😄

**Reply by TheStagnation** [reply]
Adding a comment for visibility as well. 3.5 seems miles better than 3.0, and its styles attention is pretty amazing right out of the box, better than even Flux in that aspect.

**Reply by SvenErik1968** [reply]
A user mentioned and guessed in [issue 2161](https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/2161) that the work for supporting SD 3.5 might come after **_SD 3.5 Medium_** have been released (scheduled October 29)...

**Reply by WhiteZeroX** [reply]
Now is the time for sure!

**Comment by evelryu** [comment]
Any news about this.

**Reply by SvenErik1968** [reply]
See https://github.com/lllyasviel/stable-diffusion-webui-forge/pull/2183

**Reply by evelryu** [reply]
Happy to know they are working on it.

**Comment by ArmadstheDoom** [comment]
So I know that there's a branch version that has integration, any word when it will be merged into main?

**Comment by mondermurshed** [comment]
We got no official support till now ?

**Reply by SvenErik1968** [reply]
@DenOfEquity opened a [pull request](https://github.com/lllyasviel/stable-diffusion-webui-forge/pull/2688) yesterday...

**Reply by HMRMike** [reply]
They did it!
~~Make sure to enable T5 in the webui settings for SD3, it complains otherwise~~
![image](https://github.com/user-attachments/assets/b08ba144-ec4e-4eac-8466-1407ae423519)
